### PR TITLE
Feature/get session

### PR DESCRIPTION
**Pull request by PinLin**
to solve #24

### CODE REVIEW

File: src/session/session.controller.ts [hunk 1]
```diff
@@ -1,13 +1,17 @@
-import { Controller, Post, Delete, UseGuards, Request } from '@nestjs/common';
+import { Controller, Get, Post, Delete, UseGuards, Request } from '@nestjs/common';
 import { Request as ExpressRequest } from 'express';
 import { LoginGuard } from '../common/guards/login.guard';
+import { AuthenticatedGuard } from 'src/common/guards/authenticated.guard';
```
Comment (virtualprism) [inline src/session/session.controller.ts]: 咻一下ㄅ

**Review by virtualprism — CHANGES_REQUESTED** [review]
咻咻咻一下ㄅ

File: src/session/session.controller.ts (hunk 1 above)
Comment (PinLin) [reply src/session/session.controller.ts]: 豪

**Review by virtualprism — APPROVED** [review]
Goooood

**Pull request merged**